vote/README.md

```diff
@@ -0,0 +1,2 @@
+# Voting App
+A simple distributed application running across multiple Docker containers deployed as a CNAB
```

Comment:
Perhaps mention that this bundle is Helm-based?  (Looks like there is another voting app example based on Docker Compose.)